### PR TITLE
Update README.md for expression recognition

### PR DESCRIPTION
**Pull request by rohanakut**
Modified the Readme for expression recognition

---

## Required Information

- [ ] I referenced the issue addressed in this PR.
- [ ] I described the changes made and how these address the issue.
- [ ] I described how I tested these changes.

## Coding/Commit Requirements

* [ ] I followed applicable coding standards where appropriate (e.g., [PEP8](https://pep8.org/))
* [ ] I have not committed any models or other large files.

## New Component Checklist (**mandatory** for new microservices)

* [ ] I added an entry to `docker-compose.yml` and `build.yml`.
* [ ] I created A CI workflow under `.github/workflows`.
* [ ] I have created a `README.md` file that describes what the component does and what it depends on (other microservices, ML models, etc.).

OR
* [x] I have not added a new component in this PR.

### CODE REVIEW

File: preprocessors/emotion-recognition/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Expression Recognition Preprocessor
 
-This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` 
+This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` The code is built on [Deepface](https://github.com/serengil/deepface). There are several available models such as FaceNet, Openface, DeepFace etc. The code can be modified to use either of the models. The modifications need to be performed on `line 105` to specify which model should be used for analysis.
```
Comment (jeffbl) [inline preprocessors/emotion-recognition/README.md:3]: suggest "any of the models" if you mean any of the supported ones. Either is for choosing between just two options.

Pointing to a specific line number is fragile. Perhaps give the variable name the developer can search to find this setting? Or can this be set at the top of the file, as a configuration option?

**Review by jeffbl — CHANGES_REQUESTED** [review]

File: preprocessors/emotion-recognition/README.md [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
 
-In order to run the API as a docker container use the following commands: 
+The code to use this module as an API can be found in emotion-recognition.py. This module is fully versionned, the versions of the libraries used can be found in `requirements.txt` and in the `Dockerfile`.
 
-```docker build -t <image-name>```
 
-```docker run --publish <port>:5000 <image-name>```
+It is to be noted that AI models learn human biases reflected in the datasets, leading to incorrect predictions. For example, when a popular emotion recognition model was used on NBA players, the model detected that African-American players were twice as angry compared to their white counterparts \cite{rhue_2019}. Hence such models should be used with caution as it might lead to offensive results.
```
Comment (jeffbl) [inline preprocessors/emotion-recognition/README.md:8]: If you put in a reference, it needs to be in a format that someone can actually access. Please cite appropriately, and include the full reference at the bottom of the README

**Comment by rohanakut** [comment]
@jeffbl your comments are addressed

File: preprocessors/emotion-recognition/README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # Expression Recognition Preprocessor
 
-This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` The code is built on [Deepface](https://github.com/serengil/deepface). There are several available models such as FaceNet, Openface, DeepFace etc. The code can be modified to use either of the models. The modifications need to be performed on `line 105` to specify which model should be used for analysis. 
+This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` The code is built on [Deepface](https://github.com/serengil/deepface). There are several available models such as FaceNet, Openface, DeepFace etc. The code can be modified to use any of the models. The developer needs to pass `detector_backend` argument to the `Deepface.analyze` function to specify the model. Details about the model section can be found in the [Deepface Reademe](https://github.com/serengil/deepface)
```
Comment (jeffbl) [inline preprocessors/emotion-recognition/README.md:3]: typos: "Reademe"->"Readme" and "model selection" -> "model selection"

File: preprocessors/emotion-recognition/README.md [hunk 4]
```diff
@@ -21,3 +21,6 @@ Then run the container with :
 ```bash
 docker run -d --rm --gpus all -p <port>:5000 <image-name>
 ```
+
+## References
+[1] Rhue, Lauren. "Racial influence on automated perceptions of emotions." Available at SSRN 3281765 (2018).
```
Comment (jeffbl) [inline preprocessors/emotion-recognition/README.md:26]: Add link to paper via DOI: https://dx.doi.org/10.2139/ssrn.3281765

**Comment by rohanakut** [comment]
@jeffbl please merge this once you get free. I have resolved all the comments

File: preprocessors/emotion-recognition/README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
...
+This preprocessor has been created in order to detect expressions of an individual person. The output generated by a preprocessor would be ```happy, neutral or sad.``` The code is built on [Deepface](https://github.com/serengil/deepface). There are several available models such as FaceNet, Openface, DeepFace etc. The code can be modified to use any of the models. The developer needs to pass `detector_backend` argument to the `Deepface.analyze` function to specify the model. Details about the model selection can be found in the [Deepface Readme](https://github.com/serengil/deepface)
 
 The code to use this module as an API can be found in emotion-recognition.py. This module is fully versionned, the versions of the libraries used can be found in `requirements.txt` and in the `Dockerfile`.
```
Comment (jeffbl) [inline preprocessors/emotion-recognition/README.md:5]: Another typo: "versionned". Can you please spell check documentation before PR?

**Comment by rohanakut** [comment]
@jeffbl have passed it through a spell checker and also tried to improve the wording

**Pull request merged**